notebooks/QA_evaluation_metrics_demo.ipynb

```diff
@@ -0,0 +1,1536 @@
+{
```

Comment:
<p>What if we take a metric, let's say ROGUE, find and show examples where it is minimum, max and average. Then do the same thing with others. That will give an insight into what the metric finds and how it is relevant. Once we establish that a metric should be high, we can come up with missing aspects in the answer string that'll improve the metric.</p><p>The end goal of all these analysis is to find ways to improve the methods that generate answers...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>